ballerina/tests/data_binding_publish_tests.bal

```diff
@@ -31,9 +31,28 @@ boolean receivedBooleanValuePublish = false;
 Person? receivedPersonValuePublish = ();
 map<Person>? receivedMapValuePublish = ();
 table<Person>? receivedTableValuePublish = ();
+string receivedStringPayloadValuePublish = "";
+string receivedBytesPayloadValuePublish = "";
```

Comment:
Do we need these 2 variables? Didn't see any usage.

Reply:
Missed to add these to the test cases
Fixed in https://github.com/ballerina-platform/module-ballerinax-nats/pull/427/commits/433577ffabea99c86a5ec34828b469bce521fc85